### ISSUE TITLE
Urorectal septum malformation sequence

### ISSUE THREAD

**Issue by TonyRoscioli** [obophenotype/human-phenotype-ontology#7449]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This is a synonym for [Persistent cloaca HP:0012621](https://hpo.jax.org/app/browse/term/HP:0012621)
see
Persistent cloaca (PC), alternatively referred to as urorectal septal malformation sequence (URSMS), is characterized by a common cloacal cavity that includes bladder and intestinal elements, usually with imperforate anus without exstrophy 
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5650529/

I will add this as a synonym to the term
